eclair-core/src/main/scala/fr/acinq/eclair/db/AuditDb.scala

```diff
@@ -44,11 +44,11 @@ trait AuditDb {
 
   def addPathFindingExperimentMetrics(metrics: PathFindingExperimentMetrics): Unit
 
-  def listSent(from: TimestampMilli, to: TimestampMilli): Seq[PaymentSent]
+  def listSent(from: TimestampMilli, to: TimestampMilli, paginated_opt: Option[Paginated]): Seq[PaymentSent]
```

Comment:
Can you add a default value of `None` (for all three functions) and revert the change to include `None` everywhere in `PaymentIntegrationSpec` and `AuditDbSpec`?

```suggestion
  def listSent(from: TimestampMilli, to: TimestampMilli, paginated_opt: Option[Paginated] = None): Seq[PaymentSent]
```